src/Raven.Server/Documents/Replication/OutgoingReplicationHandler.cs

```diff
@@ -208,13 +208,11 @@ private void ReplicateToDestination()
                         {
                             while (true)
                             {
-#if DEBUG
```

Comment:
We don't, we are always compiling and running in Release. All versions, 2.5/3.0/3.5/4.0.